packages/circularprogress/src/react/__stories__/index.story.tsx

```diff
@@ -1,26 +1,26 @@
 import * as core from '@pluralsight/ps-design-system-core'
-import React from 'react'
+import * as React from 'react'
 import { storiesOf } from '@storybook/react'
 
-import CircularProgress from '../index.js'
+import CircularProgress from '../../index.js'
 
 const valueStory = storiesOf('value', module)
 ;[0, 25, 50, 75, 100, 33, 66].forEach(value =>
   valueStory.add(`value ${value}`, _ => <CircularProgress value={value} />)
 )
 
 const sizeStory = storiesOf('size', module)
-Object.keys(CircularProgress.sizes).forEach(size =>
+Object.values(CircularProgress.sizes).forEach(size =>
```

Comment:
Switched to `Object.values` because `Object.keys` has a return type of `string[]`, and we want `('small' | 'medium')[]` so that it will typecheck when passing in as the prop to `CircularProgress`, which only accepts `'small' | 'medium'`

Reply:
An alternative is to do some casting